src/components/OrderConfirmation/OrderConfirmation.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { OrderItem } from '..';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+export const OrderConfirmation = ({
+  open,
+  sandwich: { name, bread, total, isHot },
+  closeModal,
+  addToOrder,
+  updateOrderOwner
+}) => {
+  const handleClose = () => {
```

Comment:
You don't really need this function as you're just calling one function. It can be passed directly on to the button 👍